README.md

```diff
@@ -1 +1,23 @@
 # react-cli
+
+This repo is an exploration for making the tooling for React that provides the best developer experience.
+
+
+## Getting started should be dead simple
+
+Something like `npm install -g react-cli` then `react init cool-project`.
+
+## Gradual integration
+
+The initial setup should be super slim, but then you can add more to it. When you start to want to test your app, you can do `react add test`, if you want to use flow then `react add flow`, wanna add crash reporting then `react add sentry`...
```

Comment:
Yeah, mb these two steps could be composed together for a better experience :+1: